Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/BankSelectorTokenizationViewModel.swift

```diff
@@ -443,11 +447,32 @@ extension BankSelectorTokenizationViewModel: UITextFieldDelegate {
 
 extension BankSelectorTokenizationViewModel {
     
-    override func handle(error: Error) {
-        ClientSession.Action.unselectPaymentMethod(resumeHandler: nil)
+    private func executeCompletionAndNullifyAfter(error: Error? = nil) {
```

Comment:
I see this iterated in all view models, you could add it in the `PaymentMethodTokenizationViewModel` class.

Reply:
✅